seqr/utils/file_utils.py

```diff
@@ -2,40 +2,68 @@
 import subprocess
 
 from seqr.utils.logging_utils import SeqrLogger
+from seqr.utils.redis_utils import safe_redis_get_json, safe_redis_set_json
 
 logger = SeqrLogger(__name__)
 
+EXPIRATION_TIME_IN_SECONDS = 3600 - 5
+GS_STORAGE_ACCESS_CACHE_KEY = 'gs_storage_access_cache_entry'
 
-def _run_command(command, user=None):
+
+def run_command(command, user=None):
```

Comment:
this is no longer used outside this file so it seems okay to make it a private method again

Reply:
After I move the get_gs_rest_api_headers() back, it is still used by igv_api.py.